routes/experimenters.js

```diff
@@ -30,3 +31,39 @@ exports.create = function (req, res) {
       });
    });
 };
+
+function updateExperimenter(req, res, exp) {
+    Experimenter.update({ _id : ObjectId(req.params.id) }, exp, function (err, numUpdated, rawRes) {
+       if (err) {
+          log.error('Error on PUT /experimenters/' + req.params.id, err);
+          return res.send(500);
+       }
+
+       if(numUpdated == 0) {
+           // no experimenters updated, bad PUT request
+           return res.send(400);
+       }
+
+       return res.status(200).send(rawRes);
+    });
+}
+
+// PUT /experimenters/:id
+exports.update = function(req, res) {
+    var exp = {};
```

Comment:
Early on, check that the request is for the user's own resource: `if (req.params.id !== req.session.userId) return res.send(401);`

(This is assuming the user id is at req.session.userId—please check whether that's the case)